FaaS/OpenFaaS/template/numpy-mp/function/set-num-threads.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
```

Comment:
Is this function folder under numpy-mp different meaning?

Reply:
no special meaning